llvm/include/llvm/IR/IntrinsicsX86.td

```diff
@@ -7572,3 +7572,42 @@ def int_x86_avx10_vfnmsub231nepbf16128 : ClangBuiltin<"__builtin_ia32_vfnmsub231
         DefaultAttrsIntrinsic<[llvm_v8bf16_ty], [llvm_v8bf16_ty, llvm_v8bf16_ty, llvm_v8bf16_ty ],
                               [IntrNoMem]>;
 }
+
+let TargetPrefix = "x86" in {
+def int_x86_avx10_vmovrsb128 : ClangBuiltin<"__builtin_ia32_vmovrsb128">,
+        Intrinsic<[llvm_v16i8_ty], [llvm_ptr_ty],
```

Comment:
DefaultAttrsIntrinsic, the same below.

Reply:
[567131c](https://github.com/llvm/llvm-project/pull/113274/commits/567131c8c265bd29dede1ad76dd1c52a4077b270)